frameworks/cassandra/build.sh

```diff
@@ -1,6 +1,36 @@
 #!/bin/bash
 set -e
 
+usage() {
+    echo "Syntax: $0 [--cli-only] [aws|local]"
+}
+
+cli_only=
```

Comment:
Shell typically uses non-empty as true, and empty as false.  I'd prefer to never use shell for build scripts but that ship seems to have sailed.